modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java

```diff
@@ -93,6 +94,8 @@ public void doCall(Project project) {
 
 						plugin =
 							workspaceExtension.getRootProjectConfigurator();
+
+						_applyPlugins(project);
```

Comment:
Since we always want this plugin to apply, let's make this the first statement inside of the `doCall` method.

Reply:
we can move it to line 88 because we must make sure that we apply the properties plugin to workspace root project instead of apply to all child project.

Reply:
@simonjhy from what I understand, this plugin needs to be applied to all projects, including all child projects. 

If you look at the [https://github.com/stevesaliman/gradle-properties-plugin#how-do-i-use-it](https://github.com/stevesaliman/gradle-properties-plugin#how-do-i-use-it), it is recommended that we apply the plugin to all projects:
```
allprojects {
    apply plugin: 'net.saliman.properties'
}
```